engine/src/main/java/org/terasology/engine/core/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -128,33 +143,34 @@ private void initGLFW() {
         GLFW.glfwWindowHint(GLFW.GLFW_COCOA_GRAPHICS_SWITCHING, GLFW.GLFW_TRUE);
         GLFW.glfwWindowHint(GLFW.GLFW_COCOA_RETINA_FRAMEBUFFER, GLFW.GLFW_FALSE);
         GLFW.glfwWindowHint(GLFW.GLFW_DEPTH_BITS, config.getPixelFormat());
```

Comment:
I needed to set the `GLFW_CLIENT_API` hint to `GLFW_NO_API` to get this running on Windows.

```suggestion
        GLFW.glfwWindowHint(GLFW.GLFW_DEPTH_BITS, config.getPixelFormat());
        GLFW.glfwWindowHint(GLFW.GLFW_CLIENT_API, GLFW.GLFW_NO_API);
```